ztp/siteconfig-generator/siteConfig/siteConfigBuilder_test.go

```diff
@@ -225,3 +264,59 @@ func Test_getExtraManifestTemplatedRoles(t *testing.T) {
 		}
 	}
 }
+
+func Test_SNOClusterSiteConfigBuild(t *testing.T) {
+	sc := SiteConfig{}
+	err := yaml.Unmarshal([]byte(siteConfigTest), &sc)
+	assert.Nil(t, err)
```

Comment:
It's better for errors to use `assert.NoError(t, err)` - If it fails, it's easier to see why.
```suggestion
	assert.NoError(t, err)
```

Note: There are a lot of other instances of this same thing through these unit tests, and related instances such as `assert.NotEqual(t, err, nill)` which should be `assert.Error(t, err)`

Reply:
done